src/patternfly/components/Breadcrumb/breadcrumb.hbs

```diff
@@ -0,0 +1,6 @@
+<nav aria-label="breadcrumb" class="pf-c-breadcrumb{{#if breadcrumb--modifier}} {{breadcrumb--modifier}}{{/if}}"
```

Comment:
would you mind moving the `aria-label` to the line below so it's own its own line?